### PR TITLE
Cdsid 1178

### PR DESCRIPTION
**Pull request by derekspalmer**

### CODE REVIEW

File: app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
   value: Option[String] = None
 )(implicit messages: Messages)
 <div class="govuk-form-group">
-   <label class="govuk-label" for="@{field.id}">@messages(label, labelArgs: _*)</label>
+   <label class="govuk-label" for="@{field.id}">@Html(messages(label, labelArgs: _*))</label>
```
Comment (ellamdav) [inline app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html]: i don't think we should be putting html in the messages file - is there another way to achieve the layout you're after?

Comment (retroredge) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html]: +1

File: test/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml_DocumentationSpec.scala [hunk 2]
```diff
@@ -125,23 +131,244 @@ class DeclarationXml_DocumentationSpec extends WordSpec with MustMatchers {
...
 
-      val xmlElement: Elem = (new DeclarationXml).fromImportDeclaration(declaration)
+      val xmlElement: Elem = DeclarationXml().fromImportDeclaration(declaration)
 
       val governmentAgencyGoodsItem = (xmlElement \ "Declaration" \ "GoodsShipment" \ "GovernmentAgencyGoodsItem")
       (governmentAgencyGoodsItem \ "AdditionalInformation").toList.length mustBe 2
     }
-  }
 
-  "Previous documents data" should {
```
Comment (retroredge) [inline test/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml_DocumentationSpec.scala:135]: Did you intend to delete the line above?

File: app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html (hunk 1 above)
Comment (ggofron) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html]: normally I would try to achieve it with css but it's kind of a temporary solution just to make input fields in one line in our temporary big form solution. Is there any big downside to this solution?

File: test/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml_DocumentationSpec.scala (hunk 2 above)
Comment (derekspalmer) [reply test/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml_DocumentationSpec.scala:135]: Agree, but should actually be: 
"Item level Previous documents data" should {

File: app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html (hunk 1 above)
Comment (ggofron) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/inputs/input_text.scala.html]: fixed

**Pull request merged**